voc/python/ast.py

```diff
@@ -2300,7 +2300,120 @@ def visit_Index(self, node):
 
     @node_visitor
     def visit_ExceptHandler(self, node):
+        builtin_exceptions = [
+            "ArithmeticError",
+            "AssertionError",
+            "AttributeError",
+            "BaseException",
+            "BlockingIOError",
+            "BrokenPipeError",
+            "BufferError",
+            "BytesWarning",
+            "ChildProcessError",
+            "ConnectionAbortedError",
+            "ConnectionError",
+            "ConnectionRefusedError",
+            "ConnectionResetError",
+            "DeprecationWarning",
+            "EOFError",
+            "Exception",
+            "FileExistsError",
+            "FileNotFoundError",
+            "FloatingPointError",
+            "FutureWarning",
+            "GeneratorExit",
+            "ImportError",
+            "ImportWarning",
+            "IndentationError",
+            "IndexError",
+            "InterruptedError",
+            "IsADirectoryError",
+            "KeyError",
+            "KeyboardInterrupt",
+            "LookupError",
+            "MemoryError",
+            "NameError",
+            "NotADirectoryError",
+            "NotImplementedError",
+            "OSError",
+            "OverflowError",
+            "PendingDeprecationWarning",
+            "PermissionError",
+            "ProcessLookupError",
+            "ReferenceError",
+            "ResourceWarning",
+            "RuntimeError",
+            "RuntimeWarning",
+            "StandardError",
+            "StopIteration",
+            "SyntaxError",
+            "SyntaxWarning",
+            "SystemError",
+            "SystemExit",
+            "TabError",
+            "TimeoutError",
+            "TypeError",
+            "UnboundLocalError",
+            "UnicodeDecodeError",
+            "UnicodeEncodeError",
+            "UnicodeError",
+            "UnicodeTranslateError",
+            "UnicodeWarning",
+            "UserWarning",
+            "ValueError",
+            "Warning",
+            "ZeroDivisionError",
+        ]
+        if node.type in builtin_exceptions:
```

Comment:
@Heisenberg815 @eliasdorneles, so I was compiling asyncio modules using the codes from this PR and I found a minor bug where voc transpilation fails when `node.type` is None. 

A potential fix would be adding `or node.type == None` condition on this line to redirect the program flow to `self._visitCoreException(node)`.

Cheers 😃